### DISCUSSION TITLE
Bot Timeout to 0?

### DISCUSSION THREAD

**Discussion by Unbounder1** [eritislami/evobot#618] [Q&A]
How could I make it so that the bot will never disconnect from a vc?

**Comment by Takiyo0** [comment]
all you have to do is just remove this code on `/include/play.js`
```
      setTimeout(function () {
        if (queue.connection.dispatcher && message.guild.me.voice.channel) return;
        queue.channel.leave();
        queue.textChannel.send(i18n.__("play.leaveChannel"));
      }, STAY_TIME * 1000);
```
https://github.com/eritislami/evobot/blob/9b37eea244e7dfeceede23fd934e8f239be3fde1/include/play.js#L24
